doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -478,12 +478,18 @@ arguments used for configuration.  Here is a table describing these emulated dev
      - Virtio GPU type device, parameters format is:
        ``virtio-gpu[,geometry=<width>x<height>+<x_off>+<y_off> | fullscreen]``
 
-       * ``<geometry>`` specifies the mode of virtual display, windowed or fullscreen. 
+       * ``<geometry>`` specifies the mode of virtual display, windowed or fullscreen.
```

Comment:
@dbkinder @amyreye  why do we have `<>` around `geometry`? To me, this makes it look like it's something to be replaced by a value, whereas it's a fixed keyword that is expected to be followed by `=` and the virtual screen parameters.

Reply:
Good catch. I fixed it in #7315.

Reply:
Thanks @amyreye !